### PR TITLE
Updates related to SSBtools updates 

### PR DESCRIPTION
**Pull request by olangsrud**

### CODE REVIEW

File: R/GaussSuppressionFromData.R [hunk 1]
```diff
@@ -144,12 +145,25 @@
 #'                   Please note that interval calculations may have a 
 #'                   different interface in future versions.
 #'                                 
+#' @param aggregatePackage Package use to preAggregate/extraAggregate.
```
Comment (vidarkl) [inline R/GaussSuppressionFromData.R:148]: Typo: use -> used

File: R/GaussSuppressionFromData.R [hunk 2]
```diff
@@ -512,19 +540,45 @@ GaussSuppressionFromData = function(data, dimVar = NULL, freqVar=NULL,
     }
     uniqueCharVar <- charVar[!(charVar %in% dVar)]
     if (length(uniqueCharVar)) {
+      #if (length(uniqueCharVar) == 1) {
+      #  charData <- aggregate(data[uniqueCharVar], data[unique(dVar)], function(x) x[1])
```
Comment (vidarkl) [inline R/GaussSuppressionFromData.R:544]: Is this code part needed?

**Review by vidarkl — APPROVED** [review]

File: R/GaussSuppressionFromData.R (hunk 1 above)
Comment (olangsrud) [reply R/GaussSuppressionFromData.R:148]: Thanks , I will change in final submission to CRAN

File: R/GaussSuppressionFromData.R (hunk 2 above)
Comment (olangsrud) [reply R/GaussSuppressionFromData.R:544]: Thanks , I will remove in final submission to CRAN

**Pull request merged**